models/index.js

```diff
@@ -24,3 +24,20 @@ exports.selectCommentsByReviewId = (review_id) => {
   WHERE review_id = $1 ORDER BY created_at DESC;`;
   return db.query(query, [review_id]).then((result) => result.rows);
 };
+
+exports.insertComment = (review_id, newComment) => {
+  const { username, body } = newComment;
+  return db
+    .query(
+      "INSERT INTO comments (body, votes, review_id, created_at, author) VALUES ($1, $2, $3, $4, $5) RETURNING *;",
+      [body, 0, review_id, new Date(), username]
+    )
+    .then(({ rows }) => rows[0])
+    .catch(() => {
```

Comment:
you shouldn't have any .catches in your model, they should exist at the end of the promise chain which is in the controller